links.php

```diff
@@ -186,7 +207,7 @@
     include("_functions.php");
     $_language->readModule('links');
     if (!ispageadmin($userID) || !isnewsadmin($userID)) {
-        die($_language->module[ 'no_access' ]);
+        echo generateAlert($_language->module[ 'no_access' ], 'alert-danger');
     }
     $linkID = $_GET[ 'linkID' ];
```

Comment:
This is wrong.
If you remove the die() you have to put the rest into an else case